metrics/metrics.go

```diff
@@ -65,6 +65,24 @@ var (
 		Help: "Duration between time round received and time round expected.",
 	})
 
+	// ClientHTTPHeartbeatSuccess measures the success rate of HTTP hearbeat randomness requests.
+	ClientHTTPHeartbeatSuccess = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "client_http_heartbeat_success",
+		Help: "Number of successful HTTP heartbeats.",
+	}, []string{"http_address"})
```

Comment:
`http_address` here should match with with the label `http` set in the `Inc()` calls above